packages/driver/src/cy/commands/cookies.ts

```diff
@@ -4,10 +4,9 @@ import Promise from 'bluebird'
 import $utils from '../../cypress/utils'
 import $errUtils from '../../cypress/error_utils'
 
-// TODO: add hostOnly to COOKIE_PROPS
 // https://github.com/cypress-io/cypress/issues/363
 // https://github.com/cypress-io/cypress/issues/17527
-const COOKIE_PROPS = 'name value path secure httpOnly expiry domain sameSite'.split(' ')
+const COOKIE_PROPS = 'name value path secure hostOnly httpOnly expiry domain sameSite'.split(' ')
```

Comment:
1. linked issues need removed

2. the change to `pickCookieProps` change cascades into the return type of `cy.getCookie()` and `cy.getCookies()` as well as `cy.clearCookie()`

4. We likely need to update `CookieOptions` for all commands for this to be validate in all places

Reply:
Good catch! 

I got the documentation up to date in https://github.com/cypress-io/cypress-documentation/pull/5071 to add the cookie option return types. I added the types to cookie options in 19c06636ef and removed the links. I don't think `clearCookie` needs to be updated even though it uses `pickCookieProps` since it nulls out the subject.

Reply:
Funny our types already included hostOnly

Reply:
It's a bit of a blur at this point because in some cases it _might_ be in the automation client, but it shouldnt be in client types yet I don't think?

Reply:
Maybe Im reading this wrong? [Cookie type](https://github.com/cypress-io/cypress/blob/feat/support-host-only-cookies/cli/types/cypress.d.ts#L6084) which is returned here https://github.com/cypress-io/cypress/blob/feat/support-host-only-cookies/cli/types/cypress.d.ts#L1433

Reply:
I may have added that. I rebased this branch quite a bit and the changeset distinction might have got lost between rebases.